src/lib389/lib389/utils.py

```diff
@@ -1460,3 +1461,140 @@ def is_fips():
         return False
 
 
+def convert_timestamp(timestamp):
+    """Convert createtimestamp to ctime: 20170405184656Z ----> Wed Apr  5 19:46:56 2017
+    :param timestamp - A timestamp from the server
+    :return - the ctime of the timestamp
+    """
+    if timestamp:
+        time_tuple = (int(timestamp[:4]), int(timestamp[4:6]), int(timestamp[6:8]),
+                      int(timestamp[8:10]), int(timestamp[10:12]), int(timestamp[12:14]),
+                      0, 0, 0)
+        secs = time.mktime(time_tuple)
+        return time.ctime(secs)
+    else:
+        return ""
+
+
+def elapsed_time(timestamp1, timestamp2):
+    """Take two timestamps from the server and get the diff
+    """
+    if timestamp1 and timestamp2:
+        time1_tuple = (int(timestamp1[:4]), int(timestamp1[4:6]), int(timestamp1[6:8]),
+                       int(timestamp1[8:10]), int(timestamp1[10:12]), int(timestamp1[12:14]),
+                       0, 0, 0)
+        time2_tuple = (int(timestamp2[:4]), int(timestamp2[4:6]), int(timestamp2[6:8]),
+                       int(timestamp2[8:10]), int(timestamp2[10:12]), int(timestamp2[12:14]),
+                       0, 0, 0)
+        time1_secs = time.mktime(time1_tuple)
+        time2_secs = time.mktime(time2_tuple)
+        elapsed_secs = time2_secs - time1_secs
```

Comment:
I've tried to run to fixup tasks on the same machine.
First has completed successfully.
But the second took more time, and fixup-status showed some weird Elapsed time...

```
[2] Task: cn=memberOf_fixup_2022-05-02T20:40:18.420486,cn=memberOf task,cn=tasks,cn=config
--------------------------------------------------------------------------------
 - Base DN:      dc=example,dc=com
 - Filter:       (uid=*)
 - Status:       
 - Started:      Tue May  3 01:40:18 2022 (20220503004018Z)
 - Elapsed Time: -17955.999051094055
 - Exit Code:    Not finished ...
```

Reply:
Should be fixed now